### PR TITLE
Update packages

### PR DESCRIPTION
**Pull request by vdegraaf**
# Meldingen

Ticket: [SIG-6206](https://gemeente-amsterdam.atlassian.net/browse/SIG-6206)

Note: 
- while update msw tests start to fail when reaching 2.4.4. Did not find the cause yet. 

[SIG-6206]: https://gemeente-amsterdam.atlassian.net/browse/SIG-6206?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Comment by alimpens** [comment]
> while update msw tests start to fail when reaching 2.4.4. Did not find the cause yet.

I ran into this as well, I think they released a broken package.

File: apps/admin/src/app/providers/i18nProvider.ts [hunk 1]
```diff
@@ -1,8 +1,202 @@
+import { TranslationMessages } from 'ra-core'
```
Comment (alimpens) [inline apps/admin/src/app/providers/i18nProvider.ts:1]:
```suggestion
import type { TranslationMessages } from 'ra-core'
```

File: package.json [hunk 2]
```diff
@@ -37,66 +37,65 @@
...
-    "@nx/jest": "19.7.2",
-    "@nx/js": "19.7.2",
-    "@nx/next": "19.7.2",
-    "@nx/react": "19.7.2",
-    "@nx/vite": "19.7.2",
-    "@nx/web": "19.7.2",
-    "@nx/workspace": "19.7.2",
+    "@babel/core": "7.25.8",
+    "@babel/preset-react": "7.25.7",
+    "@hey-api/openapi-ts": "0.53.9",
```
Comment (alimpens) [inline package.json:54]: This will break `generate-api-client`, let's leave it for now

Comment (vdegraaf) [reply package.json:54]: You mean this line? "@hey-api/openapi-ts": "0.53.9",

Comment (alimpens) [reply package.json:54]: Yep

File: apps/admin/src/app/providers/i18nProvider.ts (hunk 1 above)
Comment (vdegraaf) [reply apps/admin/src/app/providers/i18nProvider.ts:1]: Weird it doesn't throw an error. The EsLint rule is in the config.

**Pull request merged**